bin/exercise-gen.pl

```diff
@@ -25,36 +26,36 @@
   my %arg_set = map { $_ => 1 } @ARGV;
   if ( $arg_set{'--all'} ) {
     push @exercises, $_->basename
-      foreach BASE_DIR->child('exercises')->children;
+      for BASE_DIR->child('exercises')->children;
   }
   else {
     @exercises = keys %arg_set;
   }
 }
 else {
   say 'No args given; working in current directory.';
-  if ( path('.meta/exercise-data.yaml')->is_file ) {
+  if ( path( '.meta', 'exercise-data.yaml' )->is_file ) {
```

Comment:
Space inside parenthesis seems to be inconsistently applied.  See line 29 vs line 37 and others.  (Or I am not seeing the rule of when to give space inside the argument list and when not to).

Reply:
`perltidy` is responsible for this. I believe it currently does not add spacing if there is one argument contained within parens, but will add spaces if there is more than one. It would be best to change the rules in `.perltidyrc` if we want to address this. Plus if we did change it, then the same rules would likely be applied to all other perl files. Probably best for a separate commit.

Reply:
The `--perl-best--practices` option sets `--paren-tightness=1` (which also happens to be the default).

https://metacpan.org/pod/distribution/Perl-Tidy/bin/perltidy#Tightness-of-curly-braces,-parentheses,-and-square-brackets.

Reply:
I likely just did not see the consistency being based on one or more than one argument.